book.rb

```diff
@@ -0,0 +1,12 @@
+class Book
+  attr_accessor :title, :rentals
+
+  def initialize(title, _author)
+    @title = title
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] I could not find any place in the code where the `Rental` class is getting instantiated. I believe this is a good place to do so. Since we have access to the `Book` instance using `self`, we just need to pass a date and a `Person` instance to create a rental. Kindly modify the `add_rental` method as follows. 🙂️

```
def add_rental(person, date)
  Rental.new(date, self, person)
end
```

Reply:
Thanks! the change has been made